Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM ruby:2.5
+
+# fixes dpkg man page softlink error while installing postgresql-client [source: https://stackoverflow.com/a/52655008/5350059]
+RUN mkdir -p /usr/share/man/man1 && \
+    mkdir -p /usr/share/man/man7
+
+# npm is not available in Debian repo so following official instruction [source: https://github.com/nodesource/distributions/blob/master/README.md#debinstall]
+RUN curl -sL https://deb.nodesource.com/setup_10.x -o nodesource_setup.sh && \
+    bash nodesource_setup.sh && \
+    rm -f nodesource_setup.sh
+
+# install packages
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+      build-essential \
+      curl \
+      imagemagick \
```

Comment:
I wonder why imagemagick is still in the list. GPX preview image generation was moved to gd2-ffij earlier this year, and I'm not aware of other parts of the website depending on imagemagick.

Reply:
The  imagemagick requirements are still listed in the "Minimum Requirements" section of `INSTALL.md` so if they are dropped here they should probably be dropped there as well.

https://github.com/openstreetmap/openstreetmap-website/blob/master/INSTALL.md#minimum-requirements

Reply:
Right, it seems the INSTAL.md file also needs a bit of cleanup.

Reply:
@mmd-osm is there anything in the test suite which would verify whether this dependency is needed? If not, any suggestions on a suitable test I could add?

Reply:
@mmd-osm is there anything in the test suite which would verify whether this dependency is needed? If not, any suggestions on a suitable test I could add?

Reply:
I thought we got rid of imagemagick, but it seems that https://github.com/openstreetmap/openstreetmap-website/commit/6c202448ade514cd5ecca7175555011c851735e9 introduced mini_magick, which depends on imagemagick. This is only used to resize avatar images and might be a bit heavyweight for the task at hand. Maybe @tomhughes knows more.

Reply:
Well I don't believe there is any other supported approach. It only uses it via the command line though so there is no need for the development package.